docs/Quickstart/kind.md

```diff
@@ -100,7 +100,7 @@ In order to do that, edit the `/etc/hosts` file using your favourite editor and
 172.20.0.2 console.paralus.local
 ```
 
-*Refer to the value of `domain.host` in your [values.yaml](https://github.com/paralus/helm-charts/blob/main/charts/ztka/values.yaml#L145) file to find the default host.*
```

Comment:
![2022-06-27-10-46-22](https://user-images.githubusercontent.com/14259816/175865156-633d59f3-4abb-48de-9466-ebd317bee31e.png)
This line above also needs similar change.

Reply:
Done.